geonode/people/views.py

```diff
@@ -189,6 +192,50 @@ def get_queryset(self):
         queryset = self.get_serializer_class().setup_eager_loading(queryset)
         return queryset.order_by("username")
 
+    def perform_destroy(self, instance):
+        # not implemented added to make tests pass
+        if any(
```

Comment:
@RegisSinjari this was needed? if was not implemented, why was needed to make the test pass?
Consider that here we are in the API view, so this code runs always, not only for tests